timm/layers/mlp.py

```diff
@@ -133,8 +133,8 @@ def __init__(
 
     def init_weights(self):
         # override init of fc1 w/ gate portion set to weight near zero, bias=1
-        nn.init.ones_(self.fc1a.bias)
-        nn.init.normal_(self.fc1a.weight, std=1e-6)
+        nn.init.ones_(self.fc1_g.bias)
+        nn.init.normal_(self.fc1_g.weight, std=1e-6)
```

Comment:
This was a typo in the previous version of timm, right? It should have always been `fc1_g`.

Reply:
@NightMachinery yes, it was a bug that @leng-yue caught and fixed...